misc/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+
+<style>
+.node {
+  stroke: #FFF;
+  stroke-width: 1.5px;
+}
+
+.link {
+  stroke: #999;
+  stroke-opacity: .2;
+}
+</style>
+
+<body>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min.js"></script>
+
+<script>
+
+  function versionToColor(version) {
+    // red - yellow - green, 0-30
+    var colors = ["#FF0000","#FF2200","#FF3300","#FF4400","#FF5500","#FF6600","#FF7700","#FF8800","#FF9900","#FFAA00","#FFBB00","#FFCC00","#FFDD00","#FFEE00","#FFFF00","#EEFF00","#DDFF00","#CCFF00","#BBFF00","#AAFF00","#99FF00","#88FF00","#77FF00","#66FF00","#55FF00","#44FF00","#33FF00","#22FF00","#11FF00","#00FF00"];
+
+    var green = "#00933B"
+    var yellow = "#F2B50F"
+    var red = "#F90101"
+
+    console.log(colors.length)
+    var color = '#000000';
+
+    if (version) {
+      var v_arr = version.split("-");
+      var v_num = v_arr[1];
+
+      if      (v_num == '0.27.1')
+        color = red
+      else if (v_num == '0.27.2')
+        color = red
+      else if (v_num == '0.27.3')
+        color = red
+      else if (v_num == '0.27.4')
+        color = red
+      else if (v_num == '0.28.0')
+        color = yellow
+      else if (v_num == '0.28.1')
+        color = green
+      else if (v_num == '0.28.2') {
+        color = green
+      }
+
+    }
+
+    return color
+  }
+
+  function pubkeyToColor(pubkey) {
+    alpha = ["n949f75evCHwgyP4fPVgaHqNHxUVN15PsJEZ3B3HnXPcPjcZAoy7", 
+             "n9MD5h24qrQqiyBC8aeqqCWvpiBiYQ3jxSr91uiDvmrkyHRdYLUj",
+             "n9L81uNCaPgtUJfaHh89gmdvXKAmSt5Gdsw2g1iPWaPkAHW5Nm4C",
+             "n9KiYM9CgngLvtRCQHZwgC2gjpdaZcCcbt3VboxiNFcKuwFVujzS",
+             "n9LdgEtkmGB9E2h3K4Vp7iGUaKuq23Zr32ehxiU8FWY7xoxbWTSA"]
```

Comment:
These are validation public keys as opposed to the public keys returned by the crawler.